content_listing/src/main/kotlin/edu/artic/exhibitions/AllExhibitionsAdapter.kt

```diff
@@ -17,13 +18,18 @@ import kotlinx.android.synthetic.main.cell_all_exhibitions_layout.view.*
 class AllExhibitionsAdapter : AutoHolderRecyclerViewAdapter<AllExhibitionsCellViewModel>() {
 
     override fun View.onBindView(item: AllExhibitionsCellViewModel, position: Int) {
-        item.exhibitionImageUrl.subscribe {
-            GlideApp.with(context)
-                    .load(it)
-                    .error(R.drawable.placeholder_medium_square)
-                    .placeholder(R.color.placeholderBackground)
-                    .into(image)
-        }.disposedBy(item.viewDisposeBag)
+        item.exhibitionImageUrl
```

Comment:
Do we have to worry about `exhibitionImageUrl` being empty here?

Reply:
If its empty we'll display the placeholder image. So we don't need to worry.